### PR TITLE
native openshift: support tls verification

### PR DESCRIPTION
**Pull request by dustymabe**

### CODE REVIEW

**Comment by dustymabe** [comment]
Darn syntax checks.. I'll fix those.. just review for code quality in the mean time.

**Comment by dustymabe** [comment]
@rtnpro after the ssl verify PR from yesterday we'll need this PR in order to get tls verification working again for native openshift `oc new-app` deployments.

**Comment by rtnpro** [comment]
:+1:

File: atomicapp/providers/openshift.py [hunk 1]
```diff
@@ -40,6 +40,9 @@
 import logging
 logger = logging.getLogger(__name__)
 
+# If running in an openshift POD via `oc new-app`, the ca file is here
+OPENSHIFT_POD_CA_FILE = "/run/secrets/kubernetes.io/serviceaccount/ca.crt"
```
Comment (cdrage) [inline atomicapp/providers/openshift.py]: Should we keep this here on in `constants.py`?

**Comment by cdrage** [comment]
The code LGTM's. I believe I outlined my thoughts on that previous OpenShift code being located within `main.py` but :+1: that it has been moved to `openshift.py`

The tests pass :sparkles: and other than my one comment, LGTM!

File: atomicapp/providers/openshift.py (hunk 1 above)
Comment (dustymabe) [inline atomicapp/providers/openshift.py]: The question is: do we put provider specific contstants into constants.py? I'm on the fence.

**Pull request merged**